src/parsers/manifestjson.js

```diff
@@ -1088,6 +1088,44 @@ export default class ManifestJSONParser extends JSONParser {
     }
   }
 
+  getExperimentApiPaths() {
+    const { experiment_apis } = this.parsedJSON;
+
+    if (!experiment_apis) {
+      return [];
+    }
+
+    // We need to build a list of API "paths" for each registered experimental
+    // API. The data in the `manifest.json` would look like this:
+    //
+    // "experiment_apis": {
+    //   "some-name": {
+    //     "schema": "experiments/some-name/schema.json",
+    //     "parent": {
+    //       "scopes": ["addon_parent"],
+    //       "script": "experiments/some-name/api.js",
+    //       "paths": [["some", "name"]]
+    //     }
```

Comment:
@willdurand technically the "experiment_apis" can also have a "child" property for API namespace that are meant to be implemented in the child process, I'm not sure if there is any out-of-the-tree privileged extensions that is actually using that, but we do support it and there are at least a couple of privileged extensions in tree using that:
- https://searchfox.org/mozilla-central/source/browser/extensions/pictureinpicture/manifest.json#23-30
- https://searchfox.org/mozilla-central/source/browser/extensions/webcompat/manifest.json#38-45

and so it seems reasonable to silent the error also for API namespace declared as "child".

Reply:
Ah yeah, thanks for letting me know. That's something I wanted to double-check tomorrow :P